runners/direct-java/src/main/java/org/apache/beam/runners/direct/DirectOptions.java

```diff
@@ -49,7 +49,8 @@
   @Default.Boolean(true)
   @Description(
       "Controls whether the DirectRunner should ensure that all of the elements of every "
-          + "PCollection are encodable. All elements in a PCollection must be encodable.")
+          + "PCollection can be encoded by the associated Coder. The associated Coder must be able "
```

Comment:
Suggestion: "Controls whether the DirectRunner should ensure that all of the elements of every PCollection can be encoded/decoded by that PCollection's coder."